test/framework/clusterctl/clusterctl_config.go

```diff
@@ -1,5 +1,3 @@
-// +build e2e
```

Comment:
Don't we need this? @wfernandes has a different PR that adds this

Reply:
@vincepri @wfernandes 
The test framework is basically a library that CAPI is offering to external users, so IMO it should build (and run tests) like any other package

Reply:
FWIW I was adding a build tag to `docker/e2e/custom_management.go `